core/src/main/resources/runtime-properties/common-shared.properties

```diff
@@ -253,18 +253,8 @@ database.password=1234
 database.driver=org.postgresql.Driver
 
 # this connection URL assumes that it is connecting to a schema called broadleaf
+database.url=jdbc:postgresql://192.168.0.138:5432/BroadleafTestConnection
 
-# Comment out the solr.source.* above and use the following
-
- database.url=jdbc:postgresql://192.168.0.138:5432/BroadleafTestConnection
-#database.url=jdbc:postgresql://localhost:5432/broadleaf
```

Comment:
don' remove those line from the code they are for developer ease as we don't have to write those lines again if we want to use local db .